ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -0,0 +1,590 @@
+import {Component} from '@angular/core';
+import {Button} from 'app/components/buttons';
+import {InfoIcon} from 'app/components/icons';
+import {Select, TextArea, TextInput} from 'app/components/inputs';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import {TooltipTrigger} from 'app/components/popups';
+import {SpinnerOverlay} from 'app/components/spinners';
+import {cdrVersionsApi, workspacesApi} from 'app/services/swagger-fetch-clients';
+import {reactStyles} from 'app/utils';
+import {ReactWrapperBase, withCurrentWorkspace, withRouteConfigData} from 'app/utils';
+import {navigate, userProfileStore} from 'app/utils/navigation';
+import {WorkspaceUnderservedPopulation} from 'app/views/workspace-edit-underserved-population/component';
+import {CdrVersion, DataAccessLevel, Workspace} from 'generated/fetch';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+export const ResearchPurposeItems = {
+  diseaseFocusedResearch: {
+    name: 'diseaseFocusedResearch',
```

Comment:
Does this need name? If not, probably worth removing. (I can't find anywhere it is used?)